src/commands/react-native/xcode.ts

```diff
@@ -280,6 +295,41 @@ export class XCodeCommand extends Command {
     return `${process.env.CONFIGURATION_BUILD_DIR}/main.jsbundle`
   }
 
+  private getPlistValue = (propertyName: string): string | number => {
+    if (!this.infoPlistPath) {
+      throw new Error('Could not find plist path')
+    }
+
+    return parsePlist(this.infoPlistPath).getPropertyValue(propertyName)
+  }
+
+  private getReleaseVersion = (): string | null => {
+    try {
+      const releaseVersion = this.getPlistValue('CFBundleShortVersionString')
```

Comment:
I believe the iOS SDK falls back to `CFBundleVersion` if `CFBundleShortVersionString` isn't found. 
Should we apply the same logic here?

Reply:
As discussed, we can assume that if it’s not defined in the Info.plist and in MARKETING_VERSION it’s likely a mistake.